executor/split.go

```diff
@@ -60,8 +61,9 @@ type splitRegionResult struct {
 }
 
 // Open implements the Executor Open interface.
-func (e *SplitIndexRegionExec) Open(ctx context.Context) error {
-	return e.splitIndexRegion(ctx)
+func (e *SplitIndexRegionExec) Open(ctx context.Context) (err error) {
+	e.splitIdxKeys, err = e.getSplitIdxKeys()
```

Comment:
Why moving `getSplitIdxKeys()` to `Open`?

Reply:
Because we do some checks in `getSplitIdxKeys`, if put it to `Next` some test cases will be failed and I think best to put it before `Next`.